xml/obs_ag_source_publish.xml

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.1"
+ xml:id="_source_publish">
```

Comment:
Why use underscores at the beginning of IDs (repeated also for the other IDs in this file)? (Or are those auto-generated via AsciiDoctor?)

I'd suggest to just remove those.

Reply:
IIRC copy and paste ... changed